src/Microsoft.Health.Dicom.Core/Features/Query/QueryResponseBuilder.cs

```diff
@@ -139,27 +138,36 @@ public IReadOnlyCollection<DicomTag> ReturnTags
         }
     }
 
-    private void Initialize(QueryExpression queryExpression)
+    // If the target resource is All Series, then Study level attributes are also returned.
+    // If the target resource is All Instances, then Study and Series level attributes are also returned.
+    // If the target resource is Study's Instances, then Series level attributes are also returned.
+    private void Initialize(QueryExpression queryExpression, int? apiMajorVersion)
     {
+        bool useNewDefaults = false;
+        if (apiMajorVersion != null && apiMajorVersion >= 2)
+        {
+            useNewDefaults = true;
+        }
+
         switch (queryExpression.QueryResource)
         {
-            case QueryResource.AllStudies:
-                _tagsToReturn = new HashSet<DicomTag>(queryExpression.IncludeFields.All ? AllStudyTags : DefaultStudyTags);
+            case (QueryResource.AllStudies):
```

Comment:
Were the parentheses intentional?

Reply:
Left over from trying to make the switch on multiple variables. Will clean up